ghost/core/core/server/data/migrations/versions/5.45/2023-04-17-11-05-add-post-revision-author.js

```diff
@@ -0,0 +1,7 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('post_revisions', 'created_by', {
```

Comment:
Appreciate you following existing convention, but I think because these are not resources we should use `author_id` instead!

Reply:
cc @daniellockyer wdyt?

Reply:
@allouis No problemo, thats updated 👍